content/en/docs/concepts/services-networking/gateway.md

```diff
@@ -0,0 +1,207 @@
+
+See the [Gateway](https://gateway-api.sigs.k8s.io/references/spec/#gateway.networking.k8s.io/v1.Gateway)
+reference for a full definition of this API kind.
+
+### HTTPRoute {#api-kind-httproute}
+
+The HTTPRoute kind specifies routing behavior of HTTP requests from a Gateway listener to backend network
+endpoints. For a Service backend, an implementation may represent the backend network endpoint as a Service
+IP or the backing Endpoints of the Service. An HTTPRoute represents configuration that is applied to the
+underlying Gateway implementation. For example, defining a new HTTPRoute may result in configuring additional
+traffic routes into a cloud load balancer or in-cluster proxy server.
+
+A minimal HTTPRoute example:
+
+```yaml
+apiVersion: gateway.networking.k8s.io/v1
+kind: HTTPRoute
+metadata:
+  name: example-httproute
+spec:
+  parentRefs:
+  - name: example-gateway
+  hostnames:
+  - "www.example.com"
+  rules:
+  - matches:
+    - path:
+        type: PathPrefix
+        value: /login
+    backendRefs:
+    - name: example-svc
+      port: 8080
+```
+
+In this example, HTTP traffic from Gateway `example-gateway` with the Host: header set to `www.example.com`
+and the request path specified as `/login` will be routed to Service `example-svc` on port `8080`.
+
+See the [HTTPRoute](https://gateway-api.sigs.k8s.io/references/spec/#gateway.networking.k8s.io/v1.HTTPRoute)
+reference for a full definition of this API kind.
+
+## Request flow
+
+Here is a simple example of HTTP traffic being routed to a Service by using a Gateway and an HTTPRoute:
+
+{{< figure src="/docs/images/gateway-request-flow.svg" alt="A diagram that provides an example of HTTP traffic being routed to a Service by using a Gateway and an HTTPRoute" class="diagram-medium" >}}
+
+_GatewayClass is ommitted in the figure for brevity purposes._
+
+In this example, the request flow for a Gateway implemented as a reverse proxy is:
+
+1. The client starts to prepare an HTTP request for the URL `http://www.example.com`
+2. The client's DNS resolver queries for the destination name and learns a mapping to
+   one or more IP addresses associated with the Gateway.
+3. The client sends a request to the Gateway IP address; the reverse proxy receives the HTTP
+   request and uses the Host: header to match a configuration that was derived from the Gateway
+   and attached HTTPRoute.
+4. Optionally, the reverse proxy can perform request header and/or path matching based
+   on match rules of the HTTPRoute.
+5. Optionally, the reverse proxy can modify the request; for example, to add or emove headers,
+   based on filter rules of the HTTPRoute.
+6. Lastly, the reverse proxy forwards the request to one or more backends.
+
+## Conformance
+
+Gateway API covers a broad set of features and is widely implemented. This combination requires
+clear conformance definitions and tests to ensure that the API provides a consistent experience
+wherever it is used.
```

Comment:
This sentence is worded a bit odd, can we mention that we in fact have these definitions and tests and perhaps link to something that goes into more detail about conformance/conformant implementations?

Reply:
This section was much more verbose but it has been paired back based on review feedback. I will read a link to the Gateway API conformance docs.